openverse_catalog/dags/common/slack.py

```diff
@@ -70,14 +72,17 @@ class SilencedSlackNotification(TypedDict):
     """
     Configuration for a silenced Slack notification.
 
-    issue:     A link to a GitHub issue which describes why the notification
-               is silenced and tracks resolving the problem.
+    issue: A link to a GitHub issue which describes why the notification
+           is silenced and tracks resolving the problem.
     predicate: Slack notifications whose text or username contain the
                predicate will be silenced. Matching is case-insensitive.
+    task_id_pattern: A regex pattern that matches the task_id of the task
+                     that triggered the notification. (Optional)
     """
 
     issue: str
     predicate: str
+    task_id_pattern: NotRequired[str | None]
```

Comment:
TIL about NotRequired in TypedDict. Nice!

Reply:
It's a newer addition!